examples/erc20/tests/erc20.rs

```diff
@@ -18,6 +18,23 @@ const TOKEN_NAME: &str = "Test Token";
 const TOKEN_SYMBOL: &str = "TTK";
 const CAP: U256 = uint!(1_000_000_U256);
 
+// Saturday, 1 January 2000 00:00:00
+const EXPIRED_DEADLINE: U256 = uint!(946_684_800_U256);
+
+// Wednesday, 1 January 3000 00:00:00
+const FAIR_DEADLINE: U256 = uint!(32_503_680_000_U256);
```

Comment:
Should these be computed dynamically instead? Like `block::timestamp() -/+ something`?

Reply:
I do not see any strong reason for this... Besides that, we would need to query blockchain that makes tests more complicated.